src/index_intf.ml

```diff
@@ -169,6 +169,9 @@ module type S = sig
   (** [is_merging t] returns true if [t] is running a merge. Raises
       {!RO_not_allowed} if called by a read-only index. *)
 
+  val force_merge : t -> unit
```

Comment:
- Can we just call it `merge`? I think it's only called `force_merge` internally because we already had a function called `merge`.
- I think I would probably expect this to block; perhaps worth mentioning that it doesn't in the doc-comment.

Reply:
Good point, can you check if the doc is clearer now? https://github.com/mirage/index/pull/253/files#diff-7f70b3697164504072732d126e841e5a06b5456874bebb65fead19a71cd639baR176

Reply:
Looks great, thanks :+1: